drivers/scheduler/openshift/openshift.go

```diff
@@ -0,0 +1,75 @@
+package openshift
+
+import (
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	kube "github.com/portworx/torpedo/drivers/scheduler/k8s"
+)
+
+const (
+	// SchedName is the name of the kubernetes scheduler driver implementation
+	SchedName = "openshift"
+	// SnapshotParent is the parameter key for the parent of a snapshot
+	SnapshotParent = "snapshot_parent"
+	k8sPodsRootDir = "/var/lib/kubelet/pods"
```

Comment:
this isn't needed too. Check everything in the openshift driver if it can be reused from k8s driver.